packages/z2s/src/compiler.ts

```diff
@@ -123,7 +123,7 @@ function select(
 }
 
 export function limit(limit: number | undefined): SQLQuery {
-  if (!limit) {
+  if (limit === undefined) {
```

Comment:
honest question: do we care about unit testing every corner with this fuzzing technology?